ShadowDOM/explainer.md

```diff
@@ -552,12 +736,27 @@ This suggestion looks like the following:
 
 ```html
 <my-element>
-   <template shadowrootmode="open" adoptedstylesheets="/foo.css">
-       <link rel="stylesheet" href="/foo.css" noadoptedstylesheets> <!-- no-op on browsers that support adoptedstylesheets on <template> tags -->
+   <template shadowrootmode="open" shadowrootadoptedstylesheets="foo">
+       <link rel="stylesheet" href="/foo.css" noadoptedstylesheets> <!-- no-op on browsers that support shadowrootadoptedstylesheets on <template> tags -->
    </template>
 </my-element>
 ```
 
+## Future Work
+
+This proposal expands the concept of [module specifiers](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules#:~:text=The-,module%20specifier,-provides%20a%20string) to export entries into the [module map](https://html.spec.whatwg.org/multipage/webappapis.html#module-map). This concept could also apply to the `<script>` tag when [inline module scripts](https://html.spec.whatwg.org/multipage/webappapis.html#fetch-an-inline-module-script-graph) are specified, giving the ability for these scripts to [export](https://tc39.es/ecma262/#sec-exports) values, something they are not currently capable of (see [this issue](https://github.com/whatwg/html/issues/11202)).
```

Comment:
> This proposal expands the concept of [module specifiers](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules#:~:text=The-,module%20specifier,-provides%20a%20string) to export entries into the [module map](https://html.spec.whatwg.org/multipage/webappapis.html#module-map).

This sentence doesn't quite seem to fit. I'd expect it to say something like "This proposal expands the concept of [module specifiers](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules#:~:text=The-,module%20specifier,-provides%20a%20string) to allow content in `<style>` elements to create named module map entries without referencing an external file."